Meter.c

```diff
@@ -39,16 +40,18 @@ Meter* Meter_new(const struct ProcessList_* pl, int param, const MeterClass* typ
    this->pl = pl;
    this->curItems = type->maxItems;
    this->values = type->maxItems ? xCalloc(type->maxItems, sizeof(double)) : NULL;
-   this->total = type->total;
+   this->full = type->full;
    this->caption = xStrdup(type->caption);
    if (Meter_initFn(this))
       Meter_init(this);
    Meter_setMode(this, type->defaultMode);
    return this;
 }
 
+static const char* Meter_prefixes = "KMGTPEZY";
+
 int Meter_humanUnit(char* buffer, unsigned long int value, int size) {
-   const char * prefix = "KMGTPEZY";
+   const char* prefix = Meter_prefixes;
```

Comment:
Why copy this locally?

Reply:
It won't matter because compiler is smart enough to optimize this one out.